WordPress/gutenberg
# Navigation: Move manage menus button to the navigation selector

## What?
This moves the "Manage menus" button inside the Navigation Menu Selector component

## Why?
This feels like a more natural home for this link since it relates directly to the contents of the selector.

## How?
This is a small refactoring since the previous code tidyup made things cleaner.

## Testing Instructions
1. Add a navigation block
2. Check that the manage menus link is available inside the navigation menu selector
 
## Screenshots or screencast 
<img width="280" alt="Screenshot 2022-11-15 at 21 23 17" src="https://user-images.githubusercontent.com/275961/202031281-d0c8da90-48d3-4a04-87d1-6372776e9c61.png">
